01/components/TableFooter.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const TableFooter = ({data}) => {
+    const total = data.reduce((sum, item) => sum + item.price * item.quantity, 0);
+    return (
+        <section className='TableFooter'>
+            <ul>
+                <li colSpan='3'>Suma</li>
+                <li>{total} zł</li>
```

Comment:
Można by było przekazać tablicę z odpowiednią liczbą elementów - każdy element to kolumna. Jeśli wartość jej to `null` to nic nie robimy (możemy też łączyć takie wartości przez `colSpan`) jeśli jest jakaś wartość to ją wyświetalmy.